src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -727,12 +738,30 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
             scrollbarWidth
         );
 
+    /* Tooltip */
+        let tableTooltip = derivedTooltips(
+            tooltip,
+            tooltips,
+            column_conditional_tooltips,
+            column_static_tooltip,
+            virtualized,
+            tooltip_delay,
+            tooltip_duration
+        );
```

Comment:
Memoized tooltip object with resolved values -- makes it easier to manage the tooltip's lifecycle than with plain value props.